addons/mail/models/update.py

```diff
@@ -116,6 +117,11 @@ def update_notification(self, cron_mode=True):
                 self.env['ir.config_parameter'].sudo().set_param('database.expiration_date', result['enterprise_info'].get('expiration_date'), ['base.group_user'])
                 self.env['ir.config_parameter'].sudo().set_param('database.expiration_reason', result['enterprise_info'].get('expiration_reason', 'trial'), ['base.group_system'])
                 self.env['ir.config_parameter'].sudo().set_param('database.enterprise_code', result['enterprise_info'].get('enterprise_code'), ['base.group_user'])
+                if result['enterprise_info'].get('database_already_linked_subscription_url'):
```

Comment:
You should add an "else" statement where it removes the `database.already_linked_subscription_url` and `database_already_linked_subscription_url`

Otherwise the customer will always see that his database is linked to an other one when he should renew his subscription

Reply:
I removed the if and used `result[...].get(...)` so that when it is not defined, it equal to None. Because it is None, set_param will unlink the system parameter.